test/issue219.js

```diff
@@ -20,7 +20,7 @@ tape('Basic Check: pass jsdom object to output', function(t) {
   }, function(data) {
     var window = new JSDOM().window;
     t.ok(data.htmlNode instanceof window.HTMLElement, 'htmlNode is an HTMLElement');
-    t.ok(data.svgNode instanceof window.HTMLElement, 'svgNode is an HTMLElement');
-    t.ok(data.mmlNode instanceof window.HTMLElement, 'mmlNode is an HTMLElement');
+    t.ok(data.svgNode instanceof window.SVGElement, 'svgNode is an HTMLElement');
+    t.ok(data.mmlNode instanceof window.Element, 'mmlNode is an HTMLElement');
```

Comment:
Shouldn't these read `svgNode is an SVGElement` and `mmlNode is an Element` instead of `HTMLElement` for both?

Reply:
Yes. But it's merged now?

Reply:
Yes, I was adding this to remind myself to fix it in a new PR.

Reply:
Thanks for clarifying.

Reply:
I've taken care of this now.